src/main/java/by/exadel/internship/service/InternshipService.java

```diff
@@ -22,29 +22,35 @@
     private final InternshipRepository internshipRepository;
     private final InternshipMapper internShipMapper;
 
-    public GuestInternshipDTO getById(UUID uuid) {
+    public GuestInternshipDTO getById(UUID id) {
 
         MDC.put("className", InternshipService.class.getSimpleName());
-        log.info("Try to get Internships with uuid=" + uuid);
+        log.info("Try to get Internships with id=" + id);
```

Comment:
Why not using substitution approach?

log.info("Try to get Internships with id={}", id);

Reply:
done